frontend/pages/dashboard/[[id]]/index.vue

```diff
@@ -122,12 +126,13 @@ function showDashboardCreationDialog() {
   dashboardCreationControllerModal.value?.show()
 }
 
+// when we run into an error loading a dashboard keep it here to prevent an infinity loop
```

Comment:
this kinda scares me 😧

Reply:
This is something I'm hoping to fix in my other PR ^^

Reply:
perfect thx....💪